repo/packages/B/beta-portworx-jupyter-service/0/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-portworx-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
+  "scm": "https://github.com/portworx/dcos-jupyterlab-service",
+  "maintainer": "support@portworx.com",
+  "website": "https://docs.mesosphere.com/services/beta-jupyter",
+  "description": "The Portworx Jupyter Service provisions Jupyter Notebooks is an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. Uses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. Installation Documentation: https://github.com/dcos/examples/tree/master/jupyterlab",
+  "framework": true,
+  "selected": false,
+  "upgradesFrom": [
+    "1.3.0-0.35.4-beta"
```

Comment:
hi @piyush-nimbalkar since this is 1st release just leave this empty.

cc @ryadav88